modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -574,7 +577,8 @@ impl ServerModule for Wallet {
             api_endpoint! {
                 "block_count",
                 async |module: &Wallet, context, _params: ()| -> u32 {
-                    Ok(module.consensus_block_count(&mut context.dbtx()).await)
+                    // TODO: should we change this to Option?
+                    Ok(module.consensus_block_count(&mut context.dbtx()).await.unwrap_or_default())
```

Comment:
It would be the cleanest way to do it, but it shouldn't lead to any problems to leave it as-is imo. The client generally waits for non-zero consensus block height due to the finality delay.